spec/features/expense_spec.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+# spec/features/expenses_spec.rb
+
+require 'rails_helper'
+
```

Comment:
- [ ] You have written tests for all the important components, :+1:  but please note that 13 tests are currently failing in your project. Please ensure that you fix them. :pray: 
  
  ![Screenshot from 2023-08-18 00-03-35](https://github.com/touredri/my-budget/assets/92303891/150e1d8a-7be6-413c-be9b-d2edc9df6252)

Reply:
ok @Owusu-Desmond you should run again(two or three times) to see, because user data change with gem Faker.
All tests pass in my side
![Capture d’écran 2023-08-18 022600](https://github.com/touredri/my-budget/assets/112565844/72213935-72c8-4939-b8bf-6b67a9ba73e7)